src/renderer/components/watch-video-comments/watch-video-comments.css

```diff
@@ -190,4 +207,4 @@
   text-decoration: underline;
   cursor: pointer;
   color: var(--title-color);
-}
+}
```

Comment:
Please make sure files end in a newline when you add new files and don't remove it from existing files.